src/constants/message.ts

```diff
@@ -174,3 +174,16 @@ export const NO_RESULT_MESSAGE = {
   BODY1: '단어의 철자가 정확한지 확인해 보세요.',
   BODY2: '검색 옵션을 변경해 다시 검색해 보세요.',
 };
+
+export const SUMMARY = {
+  TITLE: '세 줄 요약',
+  PENDING: 'AI가 세 줄 요약을 하고 있어요!',
+  FAIL: {
+    1: '알 수 없는 오류가 발생했어요.',
+    2: '새로고침을 눌러주세요!',
+  },
+  NOT_REQUIRED: {
+    1: '본문이 너무 짧아 요약할 수 없어요.',
+    2: '전체 글을 확인해 주세요!',
+  },
```

Comment:
```suggestion
  FAIL: {
    UNKNOWN: '알 수 없는 오류가 발생했어요.',
    REFRESH(혹은 REFRESH_REQUIRED): '새로고침을 눌러주세요!',
  },
  NOT_REQUIRED: {
    TEXT_VALIDATION(혹은 SHORT_TEXT): '본문이 너무 짧아 요약할 수 없어요.',
    CHECK_TEXT: '전체 글을 확인해 주세요!',
  },
```
요 정도 생각나는거 같아요! 이외에도 취향에 따라 맞춰서 작성하시면 될거같아요!

Reply:
오오~ 직관적이네요!!!! 추천 감사합니다 ㅎㅎ 모든 네이밍에 의미있는 네이밍을 주기가 제일 힘든거 같아요...창작의 고통이랄까😣

Reply:
수정완료 [ff8cad9](https://github.com/JECT-Study/PICK-O-Client/pull/278/commits/ff8cad96ab244e1eda1dbfcb5c6444795aa20c0c)